voting/orbs/Elections/processing_vote_test.go

```diff
@@ -195,35 +220,39 @@ func TestOrbsVotingContract_processVote_CalulateStakes_DelegatorIsGuardian(t *te
 }
 
 func TestOrbsVotingContract_processVote_ValidatorsFromEthereumToState(t *testing.T) {
-	electionBlock := uint64(60000)
-	validators := [][20]byte{{0x01}, {0x02}}
+	h := newHarnessBlockBased()
+	h.electionBlock = uint64(60000)
+	v1, v2 := h.addValidator(), h.addValidator()
+	validators := make([][20]byte, 0, 2)
+	validators = append(validators, v1.address)
+	validators = append(validators, v2.address)
 
 	InServiceScope(nil, nil, func(m Mockery) {
 		_init()
 
 		// prepare
-		mockValidatorsInEthereum(m, electionBlock, validators)
-		_setCurrentElectionBlockNumber(electionBlock)
+		h.setupOrbsStateBeforeProcessMachine()
+		mockValidatorsInEthereum(m, h.electionBlock, validators)
 
 		// call
 		_readValidatorsFromEthereumToState()
 		stateValidators := _getValidators()
 
 		// assert
 		m.VerifyMocks()
-		require.EqualValues(t, len(validators), _getNumberOfValidators())
+		require.EqualValues(t, len(h.validators), _getNumberOfValidators())
```

Comment:
if you introduced a harness, you could add a function to the harness, `h.requireValidatorsInState(t, validators)` that encapsulates the logic in the following lines